### DISCUSSION TITLE
Keine Werte der Sensoren

### DISCUSSION THREAD

**Discussion by bolek1** [DrHouseIT/everHome2MQTT#3] [Allgemeine Diskussionen (General discussions)]
Hallo, habe alles nach Anleitung gemacht. Ecotracker wid per MQTT gefunden aber gibt keine Werte aus
![image](https://github.com/user-attachments/assets/cdced0f8-b1e1-474e-8684-cdc77c2c07e6)

![image](https://github.com/user-attachments/assets/80ec436b-5a20-43a7-a0fd-830b4c16f9e3)

**Comment by DrHouseIT-Daniel** [comment]
Hi, ist der Ecotracker im Moment mit dem Internet bzw. mit der Cloud verbunden?
Werden dort die aktuellen Werte angezeigt?

**Reply by bolek1** [reply]
Ja, der Ecotracker ist Verbunden und zeigt Daten an.
![image](https://github.com/user-attachments/assets/d1ade200-905e-4d2e-9e94-b78ec0efce5d)

**Reply by bolek1** [reply]
Das ist die json Asugabe
![image](https://github.com/user-attachments/assets/62ebbfd6-046d-4e86-8ba5-71b7dacc6c17)

**Reply by DrHouseIT-Daniel** [reply]
Wenn bei den einzelnen Entitäten "Nicht Verfügbar" steht, dass muss es nicht zwingend sein, dass kein Wert des entsprechenden Sensors vorhanden ist. Es können verschiedene Bedingungen parametriert werden, damit der Sensor "Verfügbar" ist. 
Schaue bitte mal in der MQTT Info, ob du herausfinden kannst, was fehlt bzw. den falschen Wert/Status hat.
 
Ich habe mal am Beispiel des Gerätes "Cloudbox" gezeigt, wie man das sehen kann.
Schaue bitte beim Config Topic bei einem der Sensoren des EcoTrackers, welche "availability" Topics parametriert sind und schaue mal, wie die Werte sind.
 
![mqtt_info](https://github.com/user-attachments/assets/e977243f-328b-4239-84cc-b5eba67ea11c)

** Noch eine Frage **

1. Hast du nur den EcoTracker ohne CloudBox?

2. Hast du die Lokale API in der Config.yaml aktiviert oder hast du nichts geändert?
Es wäre noch eine Idee die lokale API zu aktivieren.

Wenn du nur einen EcoTracker hast, dann kannst du den default-value von "false" auf "true" ändern.
```
    electricitymeter:
      ecotracker:
        local_api:
          activated:
            device_id:
              - id: 0
                value: true
            default_value: true
```
Der Bereich
```
            device_id:
              - id: 0
                value: true
```
dient nur dazu, dass du bei mehreren EcoTrackern die Einstellung individuell machen kannst.
Du könntest also den EcoTracker mit id=7 auf "false" und den mit der id=9 auf "true" setzen.

**Reply by DrHouseIT-Daniel** [reply]
Du könntest auch mal versuchen die Konfiguration zu aktualisieren.
Das wird aber nur was bringen, wenn du nach der Installation von everHome2MQTT im everHome Portal nachträglich Änderungen gemacht hast.
![IMG-20241020-WA0000.jpg](https://github.com/user-attachments/assets/3f922d3b-c912-47f5-a963-f66c249c73e7)

**Comment by DrHouseIT-Daniel** [comment]
Hallo @bolek1 
Bist du weiter gekommen?
